data/strings/strings.txt

```diff
@@ -5388,7 +5388,7 @@
 
   [pref_appearance_dark]
     comment = Settings "Appearance" category: "Dark" title, should be consistent with the pref_appearance_title translation.
-    tags = android,ios
+    tags = ios
```

Comment:
Can Light and Dark translations be reused on Android instead of the current ones for the theme? https://github.com/organicmaps/organicmaps/pull/7371